Samples/persistent_conn.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: 'Explicit host argument'
+  hosts: junos
+#  connection: local
+  connection: juniper.device.pyez
+
+  gather_facts: no
+  collections:
+    - juniper.device
+
+  tasks:
+    # Command module executed
+    - name: show version with command
+      command:
+        commands:
+          - "show version"
+      register: junos_result
+
+    # Jsnapy module executed
+    - name: 'Performa a snap_check of storage'
+      jsnapy:
+        action: "check"
+        test_files: "test_jsnapy.yaml"
+      register: response
+
+    - name: "Print the response"
+      debug:
+        var: response
+
+    # Facts module executed
+    - name: "Get facts"
+      facts:
+      register: response
+
+    # Rpc module executed
+    - name: "Test RPC"
+      rpc:
+        rpcs:
+          - "get-config"
+          - "get-software-information"
+
+    # Config module executed
+    - name: SET JUNOS CONFIG
+      config:
+        load: "set"
+        format: "set"
+        lines: 'set system login message "This is login message"'
+        comment: "comment"
+        ignore_warning: "True"
+        config_mode: private
+      register: junos_response
+
+    # System module executed
+    - name: Reboot switch after upgrade
+      system:
+        all_re: false
+        action: "reboot"
+      register: response
+
```

Comment:
I ran multiple test case. It seems that in case of persistent connection it automatically closes the connection after the playbook is finished. No need to call close-task separately.